Dianajarenga/count_zero.py

```diff
@@ -0,0 +1,27 @@
+def firstZero(A,low,high):
```

Comment:
Will binary search work if the array is not sorted? How about just starting from the beginning to the end?